components/calendar/calendar.html

```diff
@@ -52,9 +67,10 @@
         cursor: pointer;
         border-radius: var(--radius);
         border: none;
+        font-family: 'SourceSansPro', sans-serif;
```

Comment:
the month and year font-family isn't being set by default which leads to the font being Arial instead of 'SourceSansPro', sans-serif which causes the fonts to look different.